telegram/ext/_conversationhandler.py

```diff
@@ -186,6 +186,11 @@ class ConversationHandler(BaseHandler[Update, CCT]):
         conversation. For an example on nested :class:`ConversationHandler` s, see
         :any:`examples.conversationbot`.
```

Comment:
```suggestion
        :any:`examples.nestedconversationbot.html`.
```
not related to your changes but you could change this since I spotted it

Reply:
MM, this is failing.. why?

Reply:
I removed .html at the end and it works now...